dbio/util/__init__.py

```diff
@@ -459,7 +459,8 @@ def _get_jwt_from_service_account_credentials(self):
                    'email': service_credentials["client_email"],
                    'scope': ['email', 'openid', 'offline_access'],
                    self.group_claim: 'dbio',
-                   self.email_claim: service_credentials['client_email']
+                   self.email_claim: service_credentials['client_email'],
+                   self.auth0_claim: dict(groups=['service-account'])
```

Comment:
Is `self.auth0_claim` defined somewhere?

Reply:
Oh, I see, it's on line 290

Reply:
yes! ln 290